portalnet/src/overlay_service.rs

```diff
@@ -409,8 +409,12 @@ where
             // TODO: Decide default data radius, and define a constant. Or if there is an
             // associated database, then look for a radius value there.
             let node = Node::new(enr, Distance::MAX);
+            let state = match set_connected {
+                true => ConnectionState::Connected,
+                false => ConnectionState::Disconnected,
+            };
```

Comment:
It is maybe a personal preference, but I like more the following syntax:

```suggestion
            let state = if set_connected {
                    ConnectionState::Connected
                } else {
                    ConnectionState::Disconnected,
                }
            };
```